package-lock.json

```diff
@@ -2,6 +2,7 @@
 	"name": "openrct2-ride-vehicle-editor",
 	"version": "1.0.0",
 	"lockfileVersion": 3,
+	"lockfileVersion": 3,
```

Comment:
This file shouldn't be part of the PR, could you revert these changes?

Reply:
Thanks for the feedback, reverted this mistake.